frontend/src/metabase/setup/selectors.ts

```diff
@@ -128,26 +128,26 @@ export const getSteps = createSelector(
       tokenFeatures &&
       tokenFeatures["hosting"] &&
       !hasAddedPaidPlanInPreviousStep;
+
     const shouldShowDataUsageStep = !isHosted;
 
     const maybeAddStep = (step: SetupStep, condition: boolean): SetupStep[] =>
       condition ? [step] : [];
 
-    const regularSteps: SetupStep[] = [
+    const steps: SetupStep[] = [
       "welcome",
-      "language",
+      ...maybeAddStep("language", !isEmbeddingUseCase),
```

Comment:
In the next PR https://github.com/metabase/metabase/pull/63584, this language step will be removed entirely, and replaced with the language selector. This is an interim solution to prevent a broken "language" step.